app/services/difference_calculator.rb

```diff
@@ -0,0 +1,13 @@
+class DifferenceCalculator
+  def initialize(hoods, start_date: nil, end_date: Date.today)
```

Comment:
Why default `start_date` to `nil`? 
Looks like it's always got something passed in - I don't think we need default values at all?
Unless I'm misreading the changes?

Reply:
Good call!